test/test_integration.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+
+import subprocess
+import unittest
+import re
+
+class TestIntegration(unittest.TestCase):
+    def test_assertMac(self):
+        temp = subprocess.run(['osbuild', 'data/manifests/fedora-test-label-mac.json', '--output-directory=/tmp'],
+        capture_output=True, check=True)
+        output = str(temp.stdout)
+        output = re.split(',|:|\\\\n|\\\\t|\'|\\(b\'|None\\)' , output)
```

Comment:
How is `output` meant to be used?

Reply:
Yeah your right, It looks left over!

Ill remove it

Reply:
removed :smile: